app/views/govuk_publishing_components/components/_label.html.erb

```diff
@@ -15,14 +16,16 @@
 
   hint_text_css_classes = %w[govuk-hint]
   hint_text_css_classes << "govuk-radios__hint" if is_radio_label
+
+  dir = right_to_left ? "rtl" : nil
 %>
 
 <% if is_page_heading %>
   <%= tag.h1 text, class: "govuk-label-wrapper" do %>
-    <%= tag.label text, id: id, for: html_for, class: css_classes %>
+    <%= tag.label text, id: id, for: html_for, class: css_classes, dir: dir %>
   <% end %>
 <% else %>
-  <%= tag.label text, id: id, for: html_for, class: css_classes %>
+  <%= tag.label text, id: id, for: html_for, class: css_classes, dir: dir %>
 <% end %>
 
 <% if hint_text.present? %>
```

Comment:
I guess this hint text should also follow the right_to_left option.

Reply:
Yes, good point, have added that. 👍